src/utils.js

```diff
@@ -8,3 +8,31 @@ exports.findHighest = (...numbers) => {
     return highest;    
 };
 
+
+exports.findAverage = (...numbers) => {
+  let ave = 0;
+  let res = numbers.every((element) => {return Number.isInteger(element);});
+  
+  try {
+    if(numbers.length < 1) {
+      throw "This function expects at least one argument";
+    }
+    else if(!res) {
+      throw "The function expects an integer";
+    }
+    else {
+      ave = numbers.reduce((a,b) => a + b, 0) / numbers.length;
+      if(Number.isInteger(ave)) {
+        return ave;
+      }else {
+        ave = ave.toFixed(2);
+        return parseFloat(ave);
+      }
+    }
+    
+  }
+  catch (err) {
+    return err;
+  }
```

Comment:
There is no need to catch this error. An exception should be thrown and it is normal for that to happen. The code calling this function should handle that exception on its side

Reply:
ok